source/tutorials/pd-mice.rst

```diff
@@ -369,34 +375,31 @@ If you’re still unsure of the rarefaction depth, you can also use the sample s
 
 .. 1. We can't look at the days since transplant (this is a numeric column)
 .. 2. shannon. Always shannon. Shannon is a good justification for rarefaction. Just ignore the observed ASVs behind the curtain
-.. 3. suspectible has higher diversity, wild type had a shallower sequencing depth
+.. 3. susceptible has higher diversity, wild type had a shallower sequencing depth
 .. 4. we lose 8% of samples (4 samples).
 .. 5. The samples come from mouse 457, 469, 537, and 538.
 
-After we've looked through the data, we need to select a rarefaction depth. In general, rarefaction depth is a place where an analyst needs to use their discretion. Selecting a rarefaction depth is an exercise in minimizing sequence loss while maximizing the sequences retained for diversity analysis. For high biomass samples (fecal, oral, etc), a general best estimate is a rarefaction depth of no less than 1000 sequences per sample. In low biomass samples where sequencing is shallower, a lower rarefaction depth may be selected although it’s important to keep in mind that the diversity measurements on these samples will be quite noisy and the overall quality will be low.
+After we've looked through the data, we need to select a rarefaction depth. In general, selecting a rarefaction depth is a step that requires that analyst's discretion. Selecting a rarefaction depth is an exercise in minimizing sequence loss while maximizing the sequences retained for diversity analysis. For high biomass samples (fecal, oral, etc), a general best estimate is a rarefaction depth of no less than 1000 sequences per sample. In low biomass samples where sequencing is shallower, a lower rarefaction depth may be selected although it's important to keep in mind that the diversity measurements on these samples will be quite noisy and the overall quality will be low.
 
 .. checkpoint::
 
-   *Based on the current rarefaction curve and sample summary, what sequencing depth would you pick? Why?*
```

Comment:
The set up here was deliberate as something other than a question; Id like to keep the old format because it sets up the question as an independent issue and then the rest (I think/should have been) separate. Can we go back to that?